### PR TITLE
Edited topics in traditional deployment

### PR DESCRIPTION
**Pull request by philip-menzies**
docs\get_started\plan_deployment\traditional_deployment\cluster_consideration

### CODE REVIEW

**Review by biancaluna — APPROVED** [review]

File: docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 -   When you create a cluster or a cluster member, do not use spaces in the cluster name or the cluster member name.
 -   For the deployment manager and each HCL Portal node to be in the cluster, verify that each system clock is set to within 5 minutes of the others or the addNode command fails.
 
-**Restriction:** The serverName is hardcoded to HCL Portal. The serverName cannot be changed in a stand-alone environment. If you do change it, the ConfigEngine scripts do not work. For a clustered environment, see [Virtual Member Manager integration](../user_registry_consideration/plan_vmm_int.md) for options on replacing the HCL Portal JVM.
+**Restriction:** The `serverName` value is hardcoded in HCL Portal. You cannot change the `serverName` value in a stand-alone environment. If you do change it, the `ConfigEngine` scripts do not work. For a clustered environment, see [Virtual Member Manager integration](../user_registry_consideration/plan_vmm_int.md) for options on replacing the HCL Portal JVM.
```
Comment (sunilkn020) [inline docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md:18]: Kindly change HCL portal to HCL DX

**Review by sunilkn020 — COMMENTED** [review]
Hi Philips kindly check my comments

File: docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/clus_plan_http_failover.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+To verify which server is handling user requests for a session, you can view the global settings portlet. This portlet displays the node name of the server that is handling requests. If one of the servers in the cluster fails, the request is rerouted to another server in the cluster. If distributed sessions support is enabled, the new server can access session data from the database or another server instance.
 
-By default, failover support is available for HCL Portal and any portlets that are installed with the product. To take advantage of failover support with your own developed portlets, you must ensure that your portlets are properly implemented.
+By default, failover support is available for HCL Portal and any portlets that are installed with the product. To take advantage of failover support with your own developed portlets, you must ensure that your portlets are correctly implemented.
```
Comment (sunilkn020) [inline docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/clus_plan_http_failover.md:9]: Change HCL portal to HCL DX

File: docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_guidelines.md [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
 
 ## Prerequisites
 
-In WebSphere® Application Server, a cluster is composed of multiple identical copies of an application server. A cluster member is a single application server in the cluster. HCL Digital Experience is installed as an enterprise application server within the WebSphere Application Server infrastructure. All of the clustering features available within the WebSphere Application Server infrastructure are also available and apply to HCL Portal. Thus, an HCL Portal cluster is a collection of multiple HCL Portal servers that are identically configured.
+In WebSphere® Application Server, a cluster is composed of multiple identical copies of an application server. A cluster member is a single application server in the cluster. HCL Digital Experience is installed as an enterprise application server in the WebSphere Application Server infrastructure. All the clustering features available in the WebSphere Application Server infrastructure are also available and apply to HCL Portal. Thus, an HCL Portal cluster is a collection of multiple HCL Portal servers that are identically configured.
```
Comment (sunilkn020) [inline docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_guidelines.md:7]: Change HCL portal to HCL DX

File: docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_guidelines.md [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
 
 For more information, select the appropriate IBM® WebSphere Application Server Network Deployment version by logging into [https://www.hcltechsw.com/wps/portal/about/welcome](https://www.hcltechsw.com/wps/portal/about/welcome).
 
 ## Guidelines for implementing cluster environments
 
 Implement cluster environments according to the following guidelines:
 
--   The HCL Portal databases must be transferred to a supported external database, for example: DB2®, Oracle, or SQL Server.
--   You can use several approaches to configure an external web server in a clustered environment. The instructions for installing HCL Digital Experience follow WebSphere Application Server, which involves the plug-ins installation wizard to install the binary plug-in module after the cell is set up. For a complete description of the procedure for configuring an external web server in a clustered environment, refer to the following information:
+-   The HCL Portal databases must be transferred to a supported external database, for example, a DB2®, Oracle, or SQL Server.
```
Comment (sunilkn020) [inline docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_guidelines.md:15]: Change HCL portal to HCL DX

File: docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
+-   AIX® Linux™ Windows: Except for the initial setup of the cluster, HCL Portal is not supported on a managed node that is not part of a cluster.
 
-    !!! note
-        A cluster can be created which contains only one HCL Portal server, enabling a single HCL Portal server to be operational in a managed cell.
+    !!! Note
+        A cluster can be created that contains only one HCL Portal server, enabling a single HCL Portal server to operate in a managed cell.
 
--   You cannot use the Derby database. The HCL Portal databases must be transferred to a supported external database, for example: DB2®, Oracle, or SQL Server.
--   In a clustered environment, it is not possible to change settings through the Global Settings portlet or the XML configuration interface. These changes must be made by modifying the respective properties in the WebSphere® Integrated Solutions Console.
+-   You cannot use the Derby database. The HCL Portal databases must be transferred to a supported external database, for example, DB2®, Oracle, or SQL Server.
```
Comment (sunilkn020) [inline docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md:11]: Change HCL portal to HCL DX

File: docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md [hunk 6]
```diff
@@ -1,20 +1,20 @@
...
+Some limitations apply when you set up an HCL Portal cluster.
 
 -   You must install HCL Portal as a stand-alone node before you create a cluster. You cannot install HCL Portal into a managed node.
--   AIX® Linux™ Windows™: Except for the initial setup of the cluster, HCL Portal is not supported on a managed node that is not part of a cluster.
+-   AIX® Linux™ Windows: Except for the initial setup of the cluster, HCL Portal is not supported on a managed node that is not part of a cluster.
 
-    !!! note
-        A cluster can be created which contains only one HCL Portal server, enabling a single HCL Portal server to be operational in a managed cell.
+    !!! Note
+        A cluster can be created that contains only one HCL Portal server, enabling a single HCL Portal server to operate in a managed cell.
```
Comment (sunilkn020) [inline docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md:9]: Change HCL portal to HCL DX

File: docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md [hunk 7]
```diff
@@ -1,20 +1,20 @@
...
+-   You cannot use the Derby database. The HCL Portal databases must be transferred to a supported external database, for example, DB2®, Oracle, or SQL Server.
+-   In a clustered environment, you cannot change settings through the Global Settings portlet or the XML configuration interface. You must change the configuration by modifying the respective properties in the WebSphere® Integrated Solutions Console.
 -   To support search, you must install and configure a remote search service on an application server node that is not part of the cluster.
--   Administrative actions for HCL Digital Experience are immediately visible for the user who completes them. However, another user can be assured of seeing the changes only if the user logs out of HCL Digital Experience and then logs back in. This limitation applies to both cluster and non-cluster environments.
+-   Administrative actions for HCL Digital Experience are immediately visible to the user who completes them. However, another user can see the changes only after the user logs out of HCL Digital Experience and then logs back in. This limitation applies to both cluster and non-cluster environments.
```
Comment (sunilkn020) [inline docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md:14]: Change HCL Digital Experience to HCL DX

File: docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md (hunk 1 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md:18]: Use "serverName" as adjective and follow the adjective with a noun so the user knows what kind of thing is being discussed. serverName parameter. serverName setting. Whatever the correct noun is.

File: docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/clus_plan_http_failover.md (hunk 2 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/clus_plan_http_failover.md:9]: OK

File: docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md (hunk 7 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md:14]: OK

**Pull request merged**